documentation/dashboard/working_with_data.md

```diff
@@ -290,7 +290,18 @@ may download more than one year of data by making multiple requests.
 
 <div class="my-3"></div>
 
-### Create New Report
+Delete Data
+-----------
+{: .anchor}
+
+Individual forecast or observation values cannot currently be deleted from the
```

Comment:
Start with comment about how users can delete entire time series and metadata, including any requirements about deleting time series data before metadata and what limitations might exist due to data sharing (e.g. you share a site with me, I create an observation/observation values on that site, and you want to delete your site).